File: api/SalesForceApi/SalesForce.Model/Acessos/Grupo.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

[Table("CadGrupo")]
public class Grupo
{

    [Required] 
    public int CodEmpresa { get; set; }

    [Required] 
    public int CodGrupo { get; set; }

    [Required]
    [StringLength(100)]
    public string NomeGrupo { get; set; }

}

File: client/lib/models/config_acessos/grupo.dart
import 'package:json_annotation/json_annotation.dart';
part 'grupo.g.dart';

@JsonSerializable()
class Grupo {
  int codEmpresa;
  int codGrupo;
  String nomeGrupo;

  Grupo(
      {required this.codEmpresa,
      required this.codGrupo,
      required this.nomeGrupo});

  factory Grupo.fromJson(Map<String, dynamic> json) => _$GrupoFromJson(json);
  Map<String, dynamic> toJson() => _$GrupoToJson(this);

  @override
  String toString() => nomeGrupo;
}
